tests/e2e/test_rosacli_cluster.go

```diff
@@ -0,0 +1,307 @@
+package e2e
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	labels "github.com/openshift/rosa/tests/ci/labels"
+	config "github.com/openshift/rosa/tests/utils/config"
+	rosacli "github.com/openshift/rosa/tests/utils/exec/rosacli"
+)
+
+var _ = Describe("Edit cluster - ",
```

Comment:
As the case title will be combined with `Describe` and `It`, do we need `-` here?

Reply:
removed